src/com/clover/remote/client/transport/websocket/WebSocketCloudCloverTransport.ts

```diff
@@ -98,19 +83,18 @@ export class WebSocketCloudCloverTransport extends WebSocketCloverTransport {
      * false to allow another reconnect attempt to be started by a separate 'thread'.
 	 */
 	protected initialize(): void {
-
 		// Do the notification call.  This needs to happen every time we attempt to connect.
         // It COULD mean that the device gets a notification when the Cloud Pay Display is
         // already running, but this is not harmful.
         let alertEndpoint:string = Endpoints.getAlertDeviceEndpoint(this.cloverServer, this.merchantId, this.accessToken);
         let deviceContactInfo:DeviceContactInfo = new DeviceContactInfo(this.deviceId.replace(/-/g, ""), true);
 		this.httpSupport.postData(alertEndpoint,
-            function(data) { this.deviceNotificationSent(data);}.bind(this),
-            function(error) {
-                this.connectionError(this.webSocket, "Error sending alert to device." + error);
+            (data) => setTimeout(() => this.deviceNotificationSent(data), 1000),
+            (error) => {
```

Comment:
I think I askedalready, but what is the compatibility level of the `=>` syntax?

Reply:
ES2015/es6 - http://caniuse.com/#feat=arrow-functions.  Since we are using TS and our target is es5 (tsconfig.json) the transpiler will take care of making sure these get converted into something that can run on es5.